engine/src/flutter/impeller/entity/contents/text_contents_unittests.cc

```diff
@@ -163,5 +166,48 @@ TEST_P(TextContentsTest, SimpleComputeVertexData2x) {
   EXPECT_RECT_NEAR(uv_rect, Rect::MakeXYWH(0.5, 0.5, 103, 103));
 }
 
+TEST_P(TextContentsTest, MaintainsShape) {
+#ifndef FML_OS_MACOSX
+  GTEST_SKIP() << "Results aren't stable across linux and macos.";
+#endif
+
+  std::shared_ptr<TextFrame> text_frame =
+      MakeTextFrame("th", "ahem.ttf", /*font_size=*/50);
+
+  std::shared_ptr<TypographerContext> context = TypographerContextSkia::Make();
+  std::shared_ptr<GlyphAtlasContext> atlas_context =
+      context->CreateGlyphAtlasContext(GlyphAtlas::Type::kAlphaBitmap);
+  std::shared_ptr<HostBuffer> host_buffer = HostBuffer::Create(
+      GetContext()->GetResourceAllocator(), GetContext()->GetIdleWaiter());
+  ASSERT_TRUE(context && context->IsValid());
+  for (int i = 0; i <= 1000; ++i) {
+    Scalar font_scale = 0.440 + (i / 1000.0);
```

Comment:
I wouldn't expect the font to maintain the same aspect ratio across larger scales, but across small values like you have here 0.5 -> 0.51 this test seems reasonable to me. 

Are the results unstable on non mac platforms?

Reply:
oh, right. I was thinking about between uvs across scales. This makes more sense 🤦